experimental/packages/otlp-exporter-base/test/node/configuration/shared-env-configuration.test.ts

```diff
@@ -96,7 +96,7 @@ export function testSharedConfigurationFromEnvironment(
       sinon.assert.calledTwice(spyLoggerWarn);
       sinon.assert.calledWithExactly(
         spyLoggerWarn,
-        'Configuration: OTEL_EXPORTER_OTLP_METRICS_TIMEOUT is invalid, expected number greater than 0 (actual: -Infinitiy)'
+        `Unknown value '-Infinitiy' for OTEL_EXPORTER_OTLP_METRICS_TIMEOUT, expected a number, using defaults`
```

Comment:
I'm surprised at this one. I would expect `getNumberFromEnv` to pass on `'-Infinity'`:

```
> process.env.FOO = '-Infinity'
'-Infinity'
> core.getNumberFromEnv('FOO')
-Infinity
```

Reply:
Resolved in [cdebac9](https://github.com/open-telemetry/opentelemetry-js/pull/5583/commits/cdebac918c71ce96b1c115064081a5f8dec9ef42).